source/SIL.AppBuilder.Portal.Frontend/src/lib/i18n.tsx

```diff
@@ -1,5 +1,19 @@
 import { translate } from 'react-i18next';
+import { compose, mapProps, withProps } from 'recompose';
 
 export { InjectedTranslateProps as i18nProps } from 'react-i18next';
 
-export const withTranslations = translate('translations');
+const defaultNamespace = 'translations';
+export const withTranslations = translate(defaultNamespace);
+
+// reduce the number of props cascaded as a result of this HoC
```

Comment:
??

Reply:
oh I was messing around with the i18n library, cause I was getting a ton of infinite renders, and I couldn't figure out what was causing them.

I'll remove my test work here